### PR TITLE
Delete rancher-operator

### PR DESCRIPTION
**Pull request by cmurphy**
rancher-operator was merged into rancher[1] and won't be installed
separately going forward on 2.6. This deletes the charts so they won't
be erroneously caught by image-scanning.

[1] https://github.com/rancher/rancher/pull/32170

### CODE REVIEW

**Review by cbron — APPROVED** [review]

**Comment by cbron** [comment]
Will merge once Arvind approves

**Review by aiyengar2 — APPROVED** [review]
LGTM. We should also clean these up in dev-v2.6 @cmurphy

**Pull request merged**
